src/helpers/createElement.js

```diff
@@ -0,0 +1,23 @@
+export default function createElement(
+  tagName,
+  classes = [],
+  text,
```

Comment:
if you use `element.append('test')` it will add an TEXTElemet automaticly
no need to your `text` parameter